transport/transport-runtime/transport-runtime.gradle

```
@@ -13,10 +13,39 @@
 // limitations under the License.
 
 plugins {
+    id "com.google.protobuf"
     id 'firebase-library'
     id 'firebase-vendor'
 }
 
+// add a dependency on the protoc plugin's fat jar to make it available to protobuf below.
+configurations.create("protobuild")
+dependencies {
+    protobuild project(path: ":transport", configuration: "shadow")
```

```suggestion
    protobuild project(path: ":encoders:protoc-gen-firebase-encoders", configuration: "shadow")
```

This adds a dep to the required plugin and should unblock you.